### DISCUSSION TITLE
[Request] <title>my english no good... ,but i want to say this mod no way to get Amethyst shard , dripstone and moss block??

### DISCUSSION THREAD

**Discussion by yaphoe** [NovaMachina-Mods/ExNihiloSequentia#372] [Ideas]
### Is there an existing suggestion for this?

- [X] I have searched the existing suggestions

### Feature Description

my english no good... ,but i want to say this mod no way to get Amethyst shard , dripstone and moss block??

### Provided Benefit

good for survice

**Comment by remplerus** [comment]
I'm moving this to discussion as there might be more things we need to add too.
